packages/api/src/external/aws/app-config.ts

```diff
@@ -51,7 +51,7 @@ async function getCxsWithFeatureFlagEnabled(
       Config.getEnvType(),
       featureFlagName
     );
-    if (featureFlag && featureFlag.enabled && featureFlag.values) {
+    if (featureFlag && featureFlag.enabled) {
```

Comment:
If the FF exists, then the list should always exist right...? Or is this protection against misclassifying a boolean FF as a string value FF?

Reply:
It might have been a left-over from the time where we didn't have the clear distinction between the two types.